openapi.yaml

```
@@ -2477,7 +2477,7 @@ components:
                     description: Size of the file in bytes.
                   modified_timestamp:
                     type: integer
-                    description: Last modification timestamp of the file.
+                    description: Last modification timestamp of the file (in s).
```

```suggestion
                    description: Last modification timestamp of the file, in Unix time seconds.
```

Is this Unix time seconds? The others seem to mention Unix time,